FAQ "request more storage space as a VO" (#662)
hpcugent/vsc_user_docs: mkdocs/docs/HPC/FAQ.md

The `egrep` command will only let entries that match with the specified regular expression `[0-9]{3}M|[0-9]G` through, which corresponds with files that consume more than 100 MB.

> ### How can I get more storage space

Reviewer: ```suggestion
### How can I get more storage space?

> [By default](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#quota) you get 3 GB of storage space for your home directory and 25 GB in the data and scratch filesystems. It is not possible to expand these quota. However it is possible to get more storage space through a [Virtual Organisation](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#virtual-organisations), which will also give you access to the [extra directories](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#vo-directories) related to that VO. The moderator of a VO can [request more storage](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#requesting-more-storage-space) for their VO.

Reviewer: Also here, one sentence per line is helpful + some minor changes inline

```suggestion

[By default](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#quota) you get 3 GB of storage space for your home directory and 25 GB in your personal directories on both the data (`$VSC_DATA`) and scratch (`$VSC_SCRATCH`) filesystems.
It is not possible to expand the storage quota for these personal directories.

You can get more storage space through a [Virtual Organisation (VO)](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#virtual-organisations),
which will give you access to the [additional directories](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#vo-directories) in a subdirectory specific to that VO (`$VSC_DATA_VO` and `$VSC_SCRATCH_VO`).
The moderators of a VO can [request more storage](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#requesting-more-storage-space) for their VO.

The `egrep` command will only let entries that match with the specified regular expression `[0-9]{3}M|[0-9]G` through, which corresponds with files that consume more than 100 MB.

### How can I get more storage space?

> [By default](https://docs.hpc.ugent.be/running_jobs_with_input_output_data/#quota) you get 3 GB of storage space for your home directory and 25 GB in your personal directories on both the data (`$VSC_DATA`) and scratch (`$VSC_SCRATCH`) filesystems.

Reviewer: Ideally we don't link to the full URL.
I should have raised this also for the other FAQ entries, to please also fix those in this PR.

Instead, you should point to the MarkDown file, and let `mkdocs` generate internal links that are correct, like this:

```suggestion
[By default](running_jobs_with_input_output_data.md#quota) you get 3 GB of storage space for your home directory and 25 GB in your personal directories on both the data (`$VSC_DATA`) and scratch (`$VSC_SCRATCH`) filesystems.
```

(see also existing FAQ entries where this is done)
